debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/StreamingSourceIT.java

```diff
@@ -582,9 +582,6 @@ private void inconsistentSchema(EventProcessingFailureHandlingMode mode) throws
                 JdbcConnection connection = db.connect();
                 Connection jdbc = connection.connection();
                 Statement statement = jdbc.createStatement()) {
-            if (mode == null) {
```

Comment:
Why is this change necessary? It ensured that the DML was executed in the streaming phase.